docs/concepts/vega-chain/network.md

```diff
@@ -1,27 +1,178 @@
+
+If three blocks in a row are filled with spam, for example if parties continue to send substantially more than three votes on one proposal, then the number of required tokens is doubled, up to a maximum of 1600 tokens.
+
+### Spam limits: Transfers 
+Each party (public key) has a limitation on how many transfer transactions it can submit per epoch, set by the network parameter
+<NetworkParameter frontMatter={frontMatter} param="spam.protection.maxUserTransfersPerEpoch" hideValue={true} />.
+
+Once that transfer limit is reached for a key, any subsequent transactions are rejected until the epoch switches over.
+<!-- (mention in transfers area) -->
+
+### Spam limits: Staking
+Staking to any number validators is subject to minimum token requirements and a limit on the number of delegations per epoch:
+
+* Minimum required number of governance tokens associated to the voting public key is determined by the network parameter <NetworkParameter frontMatter={frontMatter} param="spam.protection.delegation.min.tokens" suffix="tokens" formatter="governanceToken" />. Nominations submitted without enough tokens are rejected.
+* Maximum number of nominations per public key, per epoch is determined by the network parameter <NetworkParameter frontMatter={frontMatter} param="spam.protection.max.delegations"/>. Once that nominations limit is reached for a key, any subsequent nominations are rejected until the epoch switches over.
+
+#### Rejection policies
+Relevant transactions (governance, transfers, and staking) that don't follow the anti-spam rules are rejected, either before the block is processed, or after, depending on the circumstances.
+
+**Pre-block rejection:** Each governance transaction is checked to ensure the public key sending it meets the spam protection requirements before the transaction enters the validators' mempool. The remaining transactions that were not added to a block are rechecked after the original block's execution, to be added to the next block. If a transaction fails the applicable spam rules set by the [parameters](#spam-protection-and-pow-parameters) below, it's not processed and is removed.
+
+**Post-block rejection:** When a transaction makes it into the block, it is still checked before the block is finalised, and can then be rejected before it's passed to the application layer. A transaction may be rejected post-block and passed the initial validation if, for example, it breaches the maximum allowed rule, but nodes aren't yet aware of how many of that party's transactions will eventually be included in a block. That can happen because a validator node doesn't know what's in the other validators' mempools until transactions reach them through gossip. When the nodes evaluate a block during its execution, all mempool and transaction information is available, so any transactions that breach those spam rules are rejected and removed from the block before they can be executed.
```

Comment:
The first half of the sentence is inaccurate - essentially mempool are not part of this.

> When the nodes evaluate a block during its execution, all mempool and transaction information is available, so any transactions that breach those spam rules are rejected and removed from the block before they can be executed.

When blocks are executed, we have all the information from before the block + all transactions already seen within the block _so any transactions that breach those spam rules are rejected and removed from the block before they can be executed._